src/Installers/Windows/AspNetCoreModule-Setup/IIS-Setup/include.wxi

```diff
@@ -17,18 +11,23 @@
     <?define SystemFolder = System64Folder ?>
     <?define SysWow64Folder = SystemFolder ?>
     <?define PlatformValue = x64 ?>
+    <?define ANCMInstallerVersion = 400 ?>
```

Comment:
We seem to be diverging from the Directory.Build.props which set installer version to 200 or 500, so I kept this the way we were before hardcoding to 400, but we need to make it at least 500 for ARM64, hence why we don't just use InstallerVersion here

Reply:
Most supported OS versions should have MSI 5.0 installed, but we've mostly stuck to 2.0 for some reason. It's possible that ANCM required features that were introduced in 4.0 which is why they default to 400, but I can't say for sure.